src/main/webapp/mentors.js

```diff
@@ -51,12 +59,72 @@ function createMentorElement(name, description) {
   mentorName.innerHTML = name;
   mentorCardBody.appendChild(mentorName);
 
+  mentorCardBody.appendChild(createGitHubLink(gitHubID));
+
   const mentorDescription = document.createElement('p');
   mentorDescription.innerHTML = description;
   mentorCardBody.appendChild(mentorDescription);
 
+  mentorCardBody.appendChild(createTagRow(interestTags));
+
+  mentorCardBody.appendChild( createProjectsRow(projectIDs));
+
   mentorCard.appendChild(mentorCardBody);
   mentorContainer.appendChild(mentorCard);
 
   return mentorContainer;
 }
+
+/**
+ * Creates a github profile button element from username.
+ * @param {String} gitHubID
+ * @return {HTMLElement} userGithubButton
+ */
+function createGitHubLink(gitHubID) {
+  let gitSrc = 'https://github.com/';
+  const gitLink = gitSrc.concat(gitHubID);
+  let userGithubButton = document.createElement('a');
+  userGithubButton.innerText = 'GitHub Profile';
+  userGithubButton.className = 'btn btn-primary';
+  userGithubButton.role = 'button';
+  userGithubButton.href = gitLink;
+  return userGithubButton;
+}
+
+/**
+ * Creates a block of interest tags.
+ * @param {String[]} interestTags
+ * @return {HTMLElement} tagDiv
+ */
+function createTagRow(interestTags) {
+  let tagDiv = document.createElement('div');
+  tagDiv.className = 'row p-3';
+  for (const tagText of interestTags) {
+    let tagElement = document.createElement('div');
+    tagElement.className = 'border border-muted text-muted mr-1 mb-1 badge';
+    tagElement.innerText = tagText;
+    tagDiv.append(tagElement);
+  }
+  return tagDiv;
+}
+
+/**
+ * Creates a block of project cards.
+ * @param {String[]} projectIDs
+ * @return {HTMLElement} projectsDiv
+ */
+function createProjectsRow(projectIDs) {
+  let projectsDiv = document.createElement('div');
+  projectsDiv.className = 'row p-3';
+  for (const projectID of projectIDs) {
+    let projElement = document.createElement('div');
+    projElement.className = 'card container card-holder col-12' +
+        ' text-center project-card p-3 m-3';
+    let cardTitleElement = document.createElement('h4');
+    cardTitleElement.className = 'card-title dark-emph';
+    cardTitleElement.innerText = projectID;
```

Comment:
Same issue as above. The project ID is a numeric identifier and is not readable. I think the assumption here is that the `projectID` is the name of the GitHub repository, which it is not.

Reply:
I'm glad we had the chance to talk about this ID comment in person :) Like we agreed, eventually, when we have concretized how we will load to and draw from datastore, there will be an in-between step here to fetch the readable project name, mentor name (and most of the personal attributes for the matter) but I am glad we decided to leave the readable strings for this iteration.

Reply:
Yeah as long as we're both on the same page we're fine. This'll work for now!